task/sast-unicode-check/0.1/sast-unicode-check.yaml

```diff
@@ -282,7 +282,7 @@ spec:
             echo "Selecting auth"
             select-oci-auth "${IMAGE_URL}" > "${HOME}/auth.json"
             echo "Attaching to ${IMAGE_URL}"
-            oras attach --no-tty --registry-config "$HOME/auth.json" --artifact-type "${MEDIA_TYPE}" "${IMAGE_URL}" "${UPLOAD_FILE}:${MEDIA_TYPE}"
+            oras attach --no-tty --registry-config "$HOME/auth.json" --artifact-type "${MEDIA_TYPE}" "${IMAGE_URL}@${IMAGE_DIGEST}" "${UPLOAD_FILE}:${MEDIA_TYPE}"
```

Comment:
Same problem here.  The parameter needs to be reintroduced in the task, in the default build pipeline template, and a new version of the task (and migration to it) will be needed.